go/worker/txnscheduler/committee/node.go

```diff
@@ -360,8 +365,29 @@ func (n *Node) worker() {
 
 	n.logger.Info("starting committee node")
 
+	// Initialize transaction scheduler's algorithm.
+	txnAlgorithm, err := txnSchedulerAlgorithm.New(
+		n.commonNode.Runtime.TxnSchedulerAlgorithm,
```

Comment:
Could put a `runtime := n.commonNode.Runtime` before this block to avoid repeating the field path multiple times.